services/cannon/src/Cannon/RabbitMqConsumerApp.hs

```diff
@@ -34,16 +35,24 @@ rabbitMQWebSocketApp uid mcid mSyncMarkerId e pendingConn = do
       Log.field "user" (idToText uid)
         . Log.field "client" (maybe "<temporary>" clientToText mcid)
 
-    runWithChannel (chan, queueInfo) = bracket openWebSocket closeWebSocket $ \wsConn ->
+    runWithChannel (chan, queueInfo) = bracket openWebSocket closeWebSocket $ \conn ->
       ( do
+          activity <- newEmptyMVar
+          let wsConn =
+                WSConnection
+                  { inner = conn,
+                    activity,
+                    activityTimeout = 30000000, -- TODO
+                    pongTimeout = 30000000 -- TODO
```

Comment:
Pending TODOs. Perhaps these value should come from configs and maybe we can write a test then?

Reply:
Something went wrong when I last rebased, it seems some commits got lost.